platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/PlatformConstructor.java

```diff
@@ -155,7 +155,7 @@ static QueueThread<ReservedSignedState> stateHashSignQueue(
      * @param addressBook                           the address book
      * @param selfId                                this node's id
      * @param preConsensusSystemTransactionManager  the manager which handles system transactions pre-consensus
```

Comment:
arg name should be `preconsensusSystemTransactionManager`

Reply:
fixed